src/udp/socket.rs

```diff
@@ -1158,4 +1162,133 @@ mod test {
         });
         unwrap!(result)
     }
+
+    fn udp_rendezvous_connect_between_natted_hosts(start_delay: Duration) {
+        let _ = env_logger::init();
```

Comment:
Is it smth we wanna do for each test case?

Reply:
I tihnk so. I can move that line into the individual test cases if you want.

Reply:
Nah, I meant for every test throughout our projects :) Not these 3 test cases specifically...